overrides/default.nix

```diff
@@ -1853,6 +1853,13 @@ lib.composeManyExtensions [
         }
       );
 
+      pycurl = super.pycurl.overridePythonAttrs (
+        old: {
+          propagatedBuildInputs = (old.propagatedBuildInputs or [ ]) ++ [ pkgs.curl ];
+          nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ pkgs.curl ];
```

Comment:
Similarly to the above comment. I believe all you need is just specify `buildInputs` and add `pkgs.curl` there.

`nativeBuildInputs` is meant for tooling that are typically tools that are needed during build time. Its purpose is clearer when you do cross compiling. So you would specify there things like `cmake`, for python it would be `setuptools`, `poetry-core` etc.

Reply:
This may need to be native for the `curl-config` script. (But still needn’t be propagated.)

Reply:
Then the solution is to add curl as both a build input and a native input.

Reply:
ah, my bad.

Reply:
Thank you for the review and explanation regarding the build inputs. I'll try to make the changes tomorrow.